Q/qr_mumps/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "qr_mumps"
+version = v"3.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://buttari.perso.enseeiht.fr/qr_mumps/releases/qr_mumps-3.0.tgz",
+                  "3308a45c49854a979ce8b23db443fff4ef01e8905445fd4b142b329f7743a5f6")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd qr_mumps-3.0/
+mkdir build
+cd build/
+cmake .. -DARITH="d;s;c;z" -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=$prefix -DQRM_ORDERING_AMD=OFF -DQRM_ORDERING_METIS=ON -DQRM_ORDERING_SCOTCH=OFF -DQRM_WITH_STARPU=OFF -DQRM_WITH_CUDA=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
+make -j${nproc}
+make install
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_gfortran_versions(platforms)
```

Comment:
Maybe
```suggestion
platforms = expand_gfortran_versions(supported_platforms())
```
?

Reply:
How can you answer so fast ? The time that I update my commit you already did the suggestion. :1st_place_medal: